### PR TITLE
fixes issue #78

### PR DESCRIPTION
**Pull request by letsmailvjkumar**
Fixes issue #78 
changed logo to favicon

### CODE REVIEW

**Comment by adityaraute** [comment]
As Agriya mentioned, you don't have to close your PRs. You simply need to update the newer changes to the same branch so that all your changes are present. Will save your effort next time.

File: assets/css/contact.css
Comment (adityaraute) [inline assets/css/contact.css:1]: Do we really need this? I'm noticing that it doesn't render very well
image

**Review by adityaraute — COMMENTED** [review]
Rest looks good, just check the `contact.css` modification, and if we really need it.

File: assets/css/contact.css
Comment (letsmailvjkumar) [reply assets/css/contact.css:1]: Is it contact page? if so we need to work on that page.

Comment (letsmailvjkumar) [reply assets/css/contact.css:1]: For phone screen it's not looking good.

**Comment by adityaraute** [comment]
Many thanks for working on this.
 Looking forward to more contributions from you!

**Pull request merged**